file-formats/sito/type/zif_aff_sito_v1.intf.abap

```diff
@@ -0,0 +1,551 @@
+    END OF ty_sit2_obj_act_t,
+
+    "! <p class="shorttext">Texts</p>
+    "! Text table for situation object actions
+    ty_sit2_obj_act_t_list TYPE STANDARD TABLE OF ty_sit2_obj_act_t WITH DEFAULT KEY,
+
+    "! <p class="shorttext">Navigations</p>
+    "! Situation object action - navigation
+    BEGIN OF ty_sit2_obj_act_nav,
+      "! <p class="shorttext">ID</p>
+      "! Navigation id
+      id TYPE c LENGTH 32,
+    END OF ty_sit2_obj_act_nav,
+
+    "! <p class="shorttext">Navigations</p>
+    "! Situation object action - navigation
+    ty_sit2_obj_act_nav_list TYPE STANDARD TABLE OF ty_sit2_obj_act_nav WITH DEFAULT KEY,
+
+    "! <p class="shorttext">Callbacks</p>
+    "! Situation object action - callback
+    BEGIN OF ty_sit2_obj_act_cb,
+      "! <p class="shorttext">ID</p>
+      "! Callback id
+      id TYPE c LENGTH 32,
+    END OF ty_sit2_obj_act_cb,
+
+    "! <p class="shorttext">Callbacks</p>
+    "! Situation object action - callback
+    ty_sit2_obj_act_cb_list TYPE STANDARD TABLE OF ty_sit2_obj_act_cb WITH DEFAULT KEY,
+
+    "! <p class="shorttext">Parameters</p>
+    "! Situation object navigation parameter
+    BEGIN OF ty_sit2_obj_nav_par,
+      "! <p class="shorttext">Name</p>
+      "! Name of navigation parameter
+      name TYPE c LENGTH 30,
+    END OF ty_sit2_obj_nav_par,
+
+    "! <p class="shorttext">Parameters</p>
+    "! Situation object navigation parameter
+    ty_sit2_obj_nav_par_list TYPE STANDARD TABLE OF ty_sit2_obj_nav_par WITH DEFAULT KEY,
+
+    "! <p class="shorttext">Parameters</p>
+    "! Situation object callback parameter
+    BEGIN OF ty_sit2_obj_cb_par,
+      "! <p class="shorttext">Name</p>
+      "! Name of callback parameter
+      name TYPE c LENGTH 30,
+    END OF ty_sit2_obj_cb_par,
+
+    "! <p class="shorttext">Parameters</p>
+    "! Situation object callback parameter
+    ty_sit2_obj_cb_par_list TYPE STANDARD TABLE OF ty_sit2_obj_cb_par WITH DEFAULT KEY,
+
+    "! <p class="shorttext">Value Help Service Providers</p>
+    "! Property of value help service for situation object
+    BEGIN OF ty_sit2_obj_vh_s_p,
+      "! <p class="shorttext">Property</p>
+      "! Name
+      property   TYPE c LENGTH 40,
+      "! <p class="shorttext">Entity Set</p>
+      "! Name
+      entity_set TYPE c LENGTH 40,
+    END OF ty_sit2_obj_vh_s_p,
+
+    "! <p class="shorttext">Value Help Service Providers</p>
+    "! Property of value help service for situation object
+    ty_sit2_obj_vh_s_p_list TYPE STANDARD TABLE OF ty_sit2_obj_vh_s_p WITH DEFAULT KEY,
+
+    "! <p class="shorttext">Texts</p>
+    "! Situation object text
+    BEGIN OF ty_sit2_object_t,
+      "! <p class="shorttext">Name</p>
+      "! Situation object name
+      name        TYPE c LENGTH 40,
+      "! <p class="shorttext">Description</p>
+      "! Description
+      description TYPE c LENGTH 255,
+    END OF ty_sit2_object_t,
+
+    "! <p class="shorttext">Texts</p>
+    "! Situation object text
```

Comment:
```suggestion
    "! Situation object texts
```

Reply:
not relevant anymore. all text entities have been changed from array to item